.github/workflows/main.yaml

```
@@ -0,0 +1,50 @@
+name: test
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        redis-version: [4, 5, 6]
+        go: ["1.11", "1.12", "1.13", "1.14", "1.15"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Memcached Service
+        uses: niden/actions-memcached@v7
+      - name: Start Redis
+        uses: supercharge/redis-github-action@1.1.0
+        with:
+          redis-version: ${{ matrix.redis-version }}
+      - uses: actions/setup-go@v1
+        with:
+          go-version: ${{ matrix.go }}
+      - run: go test -v -coverprofile=profile.cov ./...
+
+      - name: Send coverage
```

Where is the place to send the coverage?

```suggestion
      - name: Send coverage **to where?**
```

---

send coverage do coverall.io